### DISCUSSION TITLE
Add the ability to easily copy System Report information to share in your Bug Report

### DISCUSSION THREAD

**Discussion by mikesnoeren** [craftcms/cms#11598] [Ideas] [answered]
Whenever I find something that isn't working as expected I make a bug report. 
I find myself spending a lot of time on filling in the installation information, which shouldn't be the case.

Things that could see improvement:
- The Application Info and Plugins section in Utilities -> System Report are not easily copy-pasteable.
A better options would be to add a "Copy information" button or icon which copies all relevant information to your clipboard.

- The GH bug template has seperate fields for the information displayed in Utilities -> System Report -> Application Info. 
If the above gets implemented this can be replaced with one textarea field in which we can easily paste the information.

**Comment by brandonkelly** [comment, marked as answer]
If you use the Craft Support widget on your Dashboard to post the issue (**Give feedback** → **Post on GitHub**), all of those system fields will be pre-populated for you :)

**Reply by mikesnoeren** [reply]
That's great! I Wasn't aware of this feature as we generally never use the Dashboard. 
I'll keep that in mind, thanks.

**Reply by ljclark** [reply]
Also wasn't aware of this widget option - great for Craft based issues, but doesn't' really resolve/standardise the information for other plugins issues. 

Maybe a Copy to clipboard button in the header of each section as @mikesnoeren suggested would streamline things a bit and standardise issue creation across both Craft CMS issues, as well as external plugins? Main thing is that they're formatted properly when pasted. Rough mockup attached!  

![copy-data-mockup](https://user-images.githubusercontent.com/8060816/188105093-7005ad93-a235-4141-be80-51bc084eb149.png)
